tsfc/mixedelement.py

```
@@ -95,7 +96,10 @@ def tabulate(self, order, points):
         crange = numpy.cumsum(sub_cmps)
 
         for i, e in enumerate(self.elements()):
-            table = e.tabulate(order, points)
+            try:
+                table = e.tabulate(order, points, entity)
+            except TraceError as TE:
+                table = TE.zeros
```

Are you swallowing a potential error here? Why?